app/controllers/dashboard_controller.rb

```diff
@@ -46,9 +46,11 @@ def index
   end
 
   def collections_list
-    collections = Collection.includes(:owner).distinct
-    document_sets = DocumentSet.includes(:owner).distinct
-    @collections_and_document_sets = (collections + document_sets).sort { |a,b| a.title <=> b.title }
+    if user_signed_in?
+      @collections = Collection.by_user(current_user) + DocumentSet.by_user(current_user)
+    else
+      @collections = Collection.where(restricted: false)
```

Comment:
I suspect line 52 should be `@collections = Collection.where(restricted: false) + DocumentSet.where(restricted: false)`

Reply:
@benwbrum I just tried your suggestion (thanks), but it fails because DocumentSet doesn't have a `restricted` field.